lib/gds-components/character-count/CharacterCount.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * GDS component: https://design-system.service.gov.uk/components/character-count/
+ */
+
+export interface CharacterCountProps {
+  characterLimit: number;
+  id?: string;
+  numberOfCharacters: number;
+}
+
+const getRemainingCharactersMessage = (
+  numberOfCharacters: number,
+  characterLimit: number
+) => {
+  const remainingCharacters = characterLimit - numberOfCharacters;
+  const absoluteRemainingCharacters = Math.abs(remainingCharacters);
+
+  return `You have ${absoluteRemainingCharacters} character${
+    absoluteRemainingCharacters === 1 ? "" : "s"
+  } ${remainingCharacters < 0 ? "too many" : "remaining"}`;
+};
+
+export const CharacterCount = ({
+  characterLimit,
+  id,
+  numberOfCharacters,
+}: CharacterCountProps) => (
+  <div id={id} className="govuk-hint">
+    {getRemainingCharactersMessage(numberOfCharacters, characterLimit)}
```

Comment:
do we want to highlight the text in red when the count is over the limit? 
like in the GDS example?
![image](https://github.com/digital-land/local-plans-timetable/assets/98337074/b4c3610d-c6d7-4601-a5e4-d16afc324718)

Reply:
also GDS uses bold font when over the limit

Reply:
Good spots, I thought the styles had been copied so I'll take another look!

Reply:
Sorted :)